services/application.js

```diff
@@ -339,16 +339,14 @@ class Application {
 
     async sendEmailAfterApproveApplication(context, application) {
         // org owner email
-        let org = await this.organizationService.getOrganizationByID(application?.organization?._id);
         let org_owner_email = null
-        let org_owner_id = org?.owner
-        if(org_owner_id){
-            let org_owner = await this.userService.getUserByID(org_owner_id);
-            if(org_owner?.email){
-                org_owner_email = org_owner?.email
-            }
+        let orgOwner = await this.userService.getOrgOwner(application?.organization?._id)
```

Comment:
This code results in a list of org owners that looks like this: `nullorgowner@email.com ; orgowner2@email.com` this can be replaced by using `orgOwner.join(";")`